api-tests/core/strapi/document-service/resources/fixtures/author.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "id": 1,
+    "documentId": "Author1",
+    "name": "Author1-Draft",
+    "publishedAt": null,
+    "locale": null
```

Comment:
We should have a test on what happens when locale = "fr" on a non-localized content type (that is, if they disabled it, what happens?)

Reply:
veeeery good point, I believe there is nothing that would remove the previous locale value, neither delete other localitzations of the same document and only keep the default one.

I am actually going to create a new jira ticket for this if you see it appropiate!